apps/demo/lib/use-demo-data.ts

```diff
@@ -36,7 +38,9 @@ export const useDemoData = ({
 
   useEffect(() => {
     if (data && !isEdit) {
-      resolveAllData<Props, RootProps>(data, config).then(setResolvedData);
+      resolveAllData<Props, RootProps>(data, config, metadata).then(
+        setResolvedData
+      );
     }
   }, [data, isEdit]);
```

Comment:
Linting failing - metadata needs adding to this hook.

Reply:
Corrected in [1df8b99](https://github.com/measuredco/puck/pull/882/commits/1df8b99806ef1895d495d0a97008a26c1c5eed3b)